bitnami/matomo/templates/cronjob.yaml

```diff
@@ -0,0 +1,261 @@
+{{- /*
+Copyright VMware, Inc.
+SPDX-License-Identifier: APACHE-2.0
+*/}}
+
+{{- if .Values.cronjobs.enabled -}}
+apiVersion: {{ include "common.capabilities.cronjob.apiVersion" . }}
+kind: CronJob
+metadata:
+  name: {{ include "common.names.fullname" . }}-archive
+  namespace: {{ include "common.names.namespace" . | quote }}
+  labels: {{- include "common.labels.standard" . | nindent 4 }}
+    {{- if .Values.commonLabels }}
+    {{- include "common.tplvalues.render" ( dict "value" .Values.commonLabels "context" $ ) | nindent 4 }}
+    {{- end }}
+  {{- if .Values.commonAnnotations }}
+  annotations: {{- include "common.tplvalues.render" ( dict "value" .Values.commonAnnotations "context" $ ) | nindent 4 }}
+  {{- end }}
+spec:
+  schedule: {{ .Values.cronjobs.archive.schedule | quote }}
+  suspend: {{ .Values.cronjobs.archive.suspend }}
+  concurrencyPolicy: Forbid
+  jobTemplate:
+    spec:
+      template:
+        metadata:
+          labels: {{- include "common.labels.standard" . | nindent 12 }}
+            app.kubernetes.io/component: archive
+            {{- if .Values.cronjobs.archive.podLabels }}
+            {{- include "common.tplvalues.render" ( dict "value" .Values.cronjobs.archive.podLabels "context" $ ) | nindent 12 }}
+            {{- end }}
+          {{- if .Values.cronjobs.archive.podAnnotations }}
+          annotations: {{- include "common.tplvalues.render" ( dict "value" .Values.cronjobs.archive.podAnnotations "context" $ ) | nindent 12 }}
+          {{- end }}
+        spec:
+          {{- include "matomo.imagePullSecrets" . | nindent 10 }}
+          restartPolicy: OnFailure
+          initContainers:
+            {{ include "matomo.initContainers" . | nindent 12 }}
+          containers:
+            - name: {{ include "common.names.fullname" . }}-scheduled-tasks
```

Comment:
The cronjob name is `name: {{ include "common.names.fullname" . }}-archive` but the container is `{{ include "common.names.fullname" . }}-scheduled-tasks` and runs `core:run-scheduled-tasks` and viceversa in the next cronjob. Please align this properly.

Reply:
Fixed [1556933](https://github.com/bitnami/charts/pull/18754/commits/1556933995746dae478e252f55fc5a76378cda98)